cursorless-talon/src/get_grapheme_spoken_form_entries.py

```diff
@@ -32,7 +43,7 @@ def generate_lists_from_capture(capture_name) -> Iterator[str]:
     if capture_name.startswith("self."):
         capture_name = "user." + capture_name[5:]
     try:
-        rule = registry.captures[capture_name][0].rule.rule
+        rule = registry.captures[capture_name][-1].rule.rule
```

Comment:
-1/the last item is the active list and of course the one we want to use. This was actually a bug on main that would show incorrect spoken form in the tutorial.